packages/block-library/src/term-description/index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Server-side rendering of the `core/term-description` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/term-description` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Returns the filtered post content of the current post.
+ */
+function render_block_core_term_description( $attributes, $content, $block ) {
+
+	if ( is_category() || is_tag() || is_tax() ) {
```

Comment:
Would you mind reversing the check to avoid the indentation for the most code?

`if ( ! ..... ) return ''; `

Reply:
Done 👍